network/src/commonMain/kotlin/com/wire/kalium/network/api/v0/authenticated/TeamsApiV0.kt

```diff
@@ -46,6 +47,12 @@ internal open class TeamsApiV0 internal constructor(
             httpClient.get("$PATH_TEAMS/$teamId")
         }
 
+    override suspend fun whiteListedServices(teamId: TeamId, size: Int): NetworkResponse<ServiceDetailResponse> = wrapKaliumResponse {
+        httpClient.get("$PATH_TEAMS/$teamId/$PATH_SERVICES/$PATH_WHITELISTED") {
+            parameter("size", size)
```

Comment:
i did not find a way to request the next page, and the web app set the size to 100